src/components/sidebar/sidebar.scss

```diff
@@ -8,16 +8,21 @@
   top: -1px; /* Avoids pixel rounding gaps */
   bottom: -1px;
   left: 0;
+
+  // Ensures sidebar tooltips are above code panel
   z-index: 1;
   display: flex;
   width: 100%;
   transform: translateX(calc(-100% + #{$sidebar-width-closed}));
   transition: transform ease 0.4s;
+  @media (min-width: $metadata-sidebar-code-breakpoint) {
+    z-index: 3;
+  }
```

Comment:
Don't really mind but there should probably be a space above this section.

Also fyi I don't really mind but the tooltips still go below the regular metadata panel at 700-950px widths, but I don't think it matters.

Reply:
Weird, not sure why this line was missing (or that variable was defined in the wrong place) maybe a rogue merge or formatter somewhere, I'll keep an eye on it.

Yeah I think for now that case isn't very common (compared to overlapping the code panel), so we can leave it for now?